library/src/cache/updater_state.rs

```diff
@@ -133,7 +155,7 @@ impl UpdaterState {
                 if !is_file_not_found(&e) {
                     shorebird_info!("No existing state file found: {:#}, creating new state.", e);
                 }
-                Self::create_new_and_save(storage_dir, release_version, patch_public_key)
+                Self::create_new_and_save(storage_dir, release_version, patch_public_key, None)
```

Comment:
Not sure this even needs to take a None here?  I would assume this constructor would just always call generate_client_id instead?

Reply:
Fixed